docs/pages/2020_Intro_Week/sections/workflows_basic.rst

```diff
@@ -597,16 +603,18 @@ For example, you can execute:
 
 While the workflow is running, open a different terminal and check what is happening to the calculations using ``verdi process list``.
 You will see that after a few seconds the calculations are all submitted to the scheduler and can potentially run at the same time.
-Once the work chain is completed, the equation of state dictionary will be printed:
+Once the work chain is completed, you can check the result:
 
 .. code-block:: ipython
 
-    Out[1]: {'eos': <Dict: uuid: eedffd9f-c3d4-4cc8-9af5-242ede5ac23b (pk: 2937)>}
+    In [2]: result
+    Out[2]: {'eos': <Dict: uuid: eedffd9f-c3d4-4cc8-9af5-242ede5ac23b (pk: 2937)>}
 
 As a final exercise, instead of running the ``EquationOfState``, we will submit it to the daemon.
 However, in this case the work chain will need to be globally importable so the daemon can load it.
 To achieve this, the directory containing the WorkChain definition needs to be in the ``PYTHONPATH`` in order for the AiiDA daemon to find it.
-If your ``workchains.py`` sits in ``/home/max/workchains``, add a line ``export PYTHONPATH=$PYTHONPATH:/home/max`` to the ``/home/max/.virtualenvs/aiida/bin/activate`` script.
+When your ``eos_workchain.py`` is in ``/home/max/workchains``, add a line ``export PYTHONPATH=$PYTHONPATH:/home/max/workchains`` to the ``/home/max/.virtualenvs/aiida/bin/activate`` script.
+Or, if it is in your current directory:
 
 .. code-block:: bash
 
```

Comment:
I am actually confused by the `\` in the line below this comment.
Does it work fine?

Reply:
If not, note this is a replication of a previous similar mention.

Reply:
Yes, the `$` needs to be excepted within the double quotes, which I need to get `$PWD`. Maybe there is another way of doing this, but this works fine for me. :)